opendevin/runtime/utils/runtime_build.py

```diff
@@ -185,6 +196,7 @@ def build_runtime_image(
     docker_client: docker.DockerClient,
     update_source_code: bool = False,
     save_to_local_store: bool = False,  # New parameter to control saving to local store
+    extra_deps: str = '',  # whether to install extra dependencies inside the image
```

Comment:
`optional` not `whether`?
also, why not `str | none`?

Reply:
updated to `str | none`